keps/sig-storage/3751-volume-attributes-class/kep.yaml

```diff
@@ -23,11 +23,11 @@ stage: alpha
 # The most recent milestone for which work toward delivery of this KEP has been
 # done. This can be the current (upcoming) milestone, if it is being actively
 # worked on.
-latest-milestone: "1.29"
+latest-milestone: "1.30"
 
 # The milestone at which this feature was, or is targeted to be, at each stage.
 milestone:
-  alpha: "v1.29"
+  alpha: "v1.30"
```

Comment:
1.29 is the release when we first introduced VAC as an alpha feature.  I don't think this should be changed when we continue to make improvement.

Reply:
reverted